source/thisTarget.ts

```diff
@@ -79,6 +79,11 @@ const storeTabData = once(async () => {
       { cause: error },
     );
   }
+}, {
+  // If this tab is not background service worker, and it's in a prerendering state, allow subsequent calls to be made again
+  // (so that when the tab is activated, the new frame is fetched on the next call to this function).
+  // See: https://github.com/pixiebrix/pixiebrix-extension/issues/8283
+  callAgainCallBack: () => !isBackground() && "prerendering" in document && Boolean(document.prerendering)
```

Comment:
`once` is a very thin wrapper around `const value ??= getValue()`. If you need this kind of logic around it, it's easier to drop `once` and inline the logic instead.

Reply:
> Also it seems to be duplicated in the "call again callback"
It's actually slightly different parts of the skipping/refetching logic, but I agree on your other comment that we could inline the additional `once` code to make it easier to read / understand